src/effects/native/parametriceqeffect.cpp

```diff
@@ -0,0 +1,215 @@
+    lfmGain->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    lfmGain->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    lfmGain->setNeutralPointOnScale(0.5);
+    lfmGain->setDefault(0);
+    lfmGain->setMinimum(-18);
+    lfmGain->setMaximum(18); // dB
+
+    EffectManifestParameter* lfmQ = manifest.addParameter();
+    lfmQ->setId("q1");
+    lfmQ->setName(QObject::tr("Q 1"));
+    lfmQ->setDescription(QObject::tr(
+            "Controls the bandwidth of the filter.\n"
+            "A lower Q affects a wider band of frequencies,\n"
+            "a higher Q affects a narrower band of frequencies."));
+    lfmQ->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
+    lfmQ->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    lfmQ->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    lfmQ->setNeutralPointOnScale(0.5);
+    lfmQ->setDefault(1.75);
+    lfmQ->setMinimum(0.5);
+    lfmQ->setMaximum(3.0);
+
+    EffectManifestParameter* lfmCenter = manifest.addParameter();
+    lfmCenter->setId("center1");
+    lfmCenter->setName(QObject::tr("Center 1"));
+    lfmCenter->setDescription(QObject::tr("Center frequency for Filter 1, from 100 Hz to 14 kHz"));
+    lfmCenter->setControlHint(EffectManifestParameter::ControlHint::KNOB_LOGARITHMIC);
+    lfmCenter->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    lfmCenter->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    lfmCenter->setNeutralPointOnScale(0.5);
+    lfmCenter->setDefault(kDefaultCenter1);
+    lfmCenter->setMinimum(100); // 100 Hz
+    lfmCenter->setMaximum(14000); // 14 kHz
+
+    EffectManifestParameter* hfmGain = manifest.addParameter();
+    hfmGain->setId("gain2");
+    hfmGain->setName(QObject::tr("Gain 2"));
+    hfmGain->setDescription(QObject::tr("Gain for Filter 2"));
+    hfmGain->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
+    hfmGain->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    hfmGain->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    hfmGain->setNeutralPointOnScale(0.5);
+    hfmGain->setDefault(0);
+    hfmGain->setMinimum(-18);
+    hfmGain->setMaximum(18); // dB
+
+    EffectManifestParameter* hfmQ = manifest.addParameter();
+    hfmQ->setId("q2");
+    hfmQ->setName(QObject::tr("Q 2"));
+    hfmQ->setDescription(QObject::tr(
+            "Controls the bandwidth of the filter.\n"
+            "A lower Q affects a wider band of frequencies,\n"
+            "a higher Q affects a narrower band of frequencies."));
+    hfmQ->setControlHint(EffectManifestParameter::ControlHint::KNOB_LINEAR);
+    hfmQ->setSemanticHint(EffectManifestParameter::SemanticHint::UNKNOWN);
+    hfmQ->setUnitsHint(EffectManifestParameter::UnitsHint::UNKNOWN);
+    hfmQ->setNeutralPointOnScale(0.5);
+    hfmQ->setDefault(1.75);
+    hfmQ->setMinimum(0.5);
+    hfmQ->setMaximum(3.0);
```

Comment:
Please remove references to `lmf` and `hfm` from the variable names. With GitLab's [resolvable discussion threads](https://docs.gitlab.com/ce/user/discussions/#marking-a-comment-or-discussion-as-resolved) it would be easy to keep track of when I mentioned this in a previous review comment.